pylint/checkers/deprecated.py

```diff
@@ -73,6 +78,27 @@ def visit_import(self, node):
                 mod_name, class_name = name.split(".", 1)
                 self.check_deprecated_class(node, mod_name, (class_name,))
 
+    def deprecated_decorators(self) -> Container[str]:
+        """Callback returning the deprecated decorators.
+
+        Returns:
+            collections.abc.Container of deprecated decorator names.
+        """
+        # pylint: disable=no-self-use
+        return ()
+
+    @utils.check_messages(
+        "deprecated-decorator",
+    )
+    def visit_decorators(self, node):
+        """Triggered when a decorator statement is seen"""
+        if isinstance(next(node.get_children()), astroid.Call):
```

Comment:
You call `next(node.get_children())` multiple times. Without proper checks this could potentially be a cause for errors. Better would be to define it first and access it from there:
```py
children = list(node.get_children())
if children and isinstance(children[0], astroid.Call):
    ...
```

Reply:
Done.